src/HealthGPS/static_linear_model.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include "interfaces.h"
+#include "mapping.h"
+
+#include <Eigen/Dense>
+
+namespace hgps {
+
+/// @brief Defines the linear model parameters used to initialise risk factors
+struct LinearModelParams {
+    std::unordered_map<core::Identifier, double> intercepts;
+    std::unordered_map<core::Identifier, std::unordered_map<core::Identifier, double>> coefficients;
+};
+
+/// @brief Implements the static linear model type
+///
+/// @details The static model is used to initialise the virtual population.
+class StaticLinearModel final : public HierarchicalLinearModel {
+  public:
+    /// @brief Initialises a new instance of the StaticLinearModel class
+    /// @param risk_factor_names An ordered list of risk factor names
+    /// @param risk_factor_models The linear models used to initialise a person's risk factor values
+    /// @param risk_factor_cholesky The Cholesky decomposition of the risk factor correlation matrix
+    StaticLinearModel(std::vector<core::Identifier> risk_factor_names,
+                      LinearModelParams risk_factor_models, Eigen::MatrixXd risk_factor_cholesky);
+
```

Comment:
This also throws and exception if the arguments are invalid, right?

Reply:
Correct. I'll add the throws docstring.